src/stores/mainStore.ts

```diff
@@ -112,11 +107,15 @@ export class MainStore {
                 setTimeout(() => {
                   this.setWinner();
                 }, 2000);
-                // this.setWinner();
                 setTimeout(() => {
-                  this.game.setGameStage(GameStage.Continue);
+                  if (this.player.playerMoney <= 0) {
```

Comment:
We're talking about money here. You used `<`, so player can go into debts? 😆 

It is better to handle bets properly and do not allow player place bets and lose more than he/she can afford